infrastructure/ci/dev-platform/frontend-template.yml

```
@@ -339,6 +339,196 @@ Resources:
         Fn::ImportValue:
           !Sub ${VpcStackName}-VpcId
 
+
+  frontendWebAcl:
```

```suggestion
  FrontendWebAcl:
```
And the same for the other new resources to use the same naming convention as we use in the rest of the template

---

fixed